src/dashboards/videos/details.jsonnet

```diff
@@ -119,3 +136,131 @@ dashboard.new(
   ),
   gridPos={ x: 0, y: 0, w: 12, h: 9 }
 )
+.addPanel(
+  statPanel.new(
+    title='Views count by time threshold',
+    // It would be nice to use markdown inline code blocks but they appear as regular code blocks.
+    // This is fixed in v8.0.0-beta1 of grafana.
+    // See https://github.com/grafana/grafana/releases/tag/v8.0.0-beta1
```

Comment:
Can you upgrade grafana release in this PR and also address this issue?

Reply:
Good Idea) Thanks) Updated this part)